notebooks/process_demetr_results.ipynb

```diff
@@ -0,0 +1,3162 @@
+{
```

Comment:
<p>Apply to everything:</p><ul><li>BLEU should be capitalised.</li><li>Probably a case for only including which of ChrF1 and ChrF2 performs best (on average).</li><li>Consider whether any other bar plots are easier to digest as tables.</li></ul><p> </p><p>Apply to this plot:</p><ul><li>It's more along the lines of the DEMETR sensitivity analysis but there's a discussion point here about what trend would be desirable, which I think is for  critical &gt; major &gt; minor in terms of accuracy/sensitivity, which none of them have.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/ARC-m4st/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>